### ISSUE TITLE
Add a video tutorial for setting up environment and running nodes locally using BDLS

### ISSUE THREAD

**Issue by sadityakumar9211** [BDLS-bft/experiment-guide#14]
## WHAT
SSIA

## WHY
To make the onboarding of new contributors easier. 

## HOW
Following either [cloud setup](https://github.com/BDLS-bft/experiment-guide/blob/main/tutorials/google-cloud-vm.md) or [local setup](https://github.com/BDLS-bft/experiment-guide/blob/main/docs/getting-started-nanobash.md) guide.

**Comment by sadityakumar9211** [comment]
Hey @ahmed82!
I feel I should go ahead and add a video tutorial for easier onboarding for new contributors. I have some free time and work on this during this weekend. Let me know wdyt!

**Comment by ahmed82** [comment]
Yes, @sadityakumar9211, that will be helpful for new contributors. Feel free to reach out if you need anything.
